File: TestFile.py
#!/usr/bin/python
import numpy as np
dt = 5.000000e-16
dz = 2.500000e-08
dx = 1.000000e-06
TestFile_ExtIn = np.reshape(np.fromfile("TestFile_ExtIn.dat",dtype=np.double)[0:1048576],(1024,512,2),order='F')
TestFile_ExtOut = np.reshape(np.fromfile("TestFile_ExtOut.dat",dtype=np.double)[0:1048576],(1024,512,2),order='F')

File: TestFile.m
fid = fopen('TestFile_ExtIn.dat','rb'); 
TestFile_ExtIn = fread(fid, 1048576, 'double'); 
TestFile_ExtIn = reshape(TestFile_ExtIn,[1024 512 2]); 
fclose(fid); 
fid = fopen('TestFile_ExtOut.dat','rb'); 
TestFile_ExtOut = fread(fid, 1048576, 'double'); 
TestFile_ExtOut = reshape(TestFile_ExtOut,[1024 512 2]); 
fclose(fid); 
